### PR TITLE
Feature/modifier

### PR DESCRIPTION
**Pull request by JvJefke**

### CODE REVIEW

File: src/proxy.js [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+
+	if (!onlyUsePrefix) {
+		routes.push({
+			target: "files/",
+			route: [
+				"/files",
+				"/file",
+				prefix + "/files",
+				prefix + "/file",
+				"/api/1.0.0/files",
```
Comment (ThomasBormans) [inline src/proxy.js]: `1.0.0` should be dynamic.

File: test/modifier.js [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+var sinonChai = require("sinon-chai");
+var expect = chai.expect;
+
+chai.should();
+chai.use(sinonChai);
+
+var proxyModifier = rewire("src/modifier");
+
+describe("WCMProxyHelper -- UNIT -- Modifier |", function() {
+	// var requestMock = {
```
Comment (ThomasBormans) [inline test/modifier.js]: This can be removed I guess?

**Comment by ThomasBormans** [comment]
I would also suggest adding a `.eslintignore` with the following content:
```
coverage/**
```
When the above file is added, there are only 7 problems left to fix.

**Comment by ThomasBormans** [comment]
Sorry, did not mean to close it 🙃

File: src/proxy.js (hunk 1 above)
Comment (JvJefke) [inline src/proxy.js]: The 1.0.0 is used fro legacy support. New images will always have '/files...'.

File: test/modifier.js (hunk 2 above)
Comment (JvJefke) [inline test/modifier.js]: fixed

**Pull request merged**